x/revenue/v1/keeper/evm_hooks.go

```diff
@@ -49,39 +49,50 @@ func (k Keeper) PostTxProcessing(
 	}
 
 	contract := msg.To()
+	// TODO: contract here is always nil when calling the precompile
 	if contract == nil {
 		return nil
 	}
```

Comment:
Building an ETH transaction with `msg.To` as the precompile address always comes up to `nil` here?